src/fundus/scraping/pipeline.py

```diff
@@ -53,9 +169,44 @@ def crawl(
         restrict_sources_to: Optional[List[Type[URLSource]]] = None,
         error_handling: Literal["suppress", "catch", "raise"] = "suppress",
         only_complete: Union[bool, ExtractionFilter] = True,
-        batch_size: int = 10,
+        delay: Optional[Union[float, Delay]] = None,
+        url_filter: Optional[URLFilter] = None,
+        only_unique: bool = True,
     ) -> Iterator[Article]:
+        """Yields articles from initialized publishers
+
+        Args:
+            max_articles (Optional[int]): Number of articles to retrieve. If there are fewer articles
+                than max_articles the Iterator will stop before max_articles. If None, all retrievable
+                articles are returned. Defaults to None.
```

Comment:
The wording "Retrieve" can probably be optimized. I would prefer download or crawl

Reply:
How about 'receive'? I think that's what I originally wanted to communicate here.

Reply:
In my opinion, receive suffers from the same problem of missing consistency. We should aim for consistent language across the project, especially in the key concepts. This is an argument for either download or crawl.

Reply:
Done.